### DISCUSSION TITLE
Trying to execute run.bat, but I'm receiving a early error RuntimeError: Torch is not able to use GPU;

### DISCUSSION THREAD

**Discussion by greathiago** [lllyasviel/stable-diffusion-webui-forge#241] [Q&A]
I followed the steps for 2 click installation as written, but receiving that error
My GPU is a NVIDIA GTX 1650 with 4 VRAM that works just fine with A1111 (older version). What do I do?

Full console:
```Python 3.10.6 (tags/v3.10.6:9c7b4bd, Aug  1 2022, 21:53:49) [MSC v.1932 64 bit (AMD64)]
Version: f0.0.12-latest-151-g4777898a
Commit hash: 4777898a0cd56e82f9c7cb4d0bd9122dd8b55ea4
Traceback (most recent call last):
  File "C:\Users\user\Documents\Github\webui_forge\webui\launch.py", line 51, in <module>
    main()
  File "C:\Users\user\Documents\Github\webui_forge\webui\launch.py", line 39, in main
    prepare_environment()
  File "C:\Users\user\Documents\Github\webui_forge\webui\modules\launch_utils.py", line 431, in prepare_environment
    raise RuntimeError(
RuntimeError: Torch is not able to use GPU; add --skip-torch-cuda-test to COMMANDLINE_ARGS variable to disable this check

**Comment by huchenlei** [comment]
Please attach full console logs. Thanks!

**Reply by greathiago** [reply]
Sorry
```Python 3.10.6 (tags/v3.10.6:9c7b4bd, Aug  1 2022, 21:53:49) [MSC v.1932 64 bit (AMD64)]
Version: f0.0.12-latest-151-g4777898a
Commit hash: 4777898a0cd56e82f9c7cb4d0bd9122dd8b55ea4
Traceback (most recent call last):
  File "C:\Users\user\Documents\Github\webui_forge\webui\launch.py", line 51, in <module>
    main()
  File "C:\Users\user\Documents\Github\webui_forge\webui\launch.py", line 39, in main
    prepare_environment()
  File "C:\Users\user\Documents\Github\webui_forge\webui\modules\launch_utils.py", line 431, in prepare_environment
    raise RuntimeError(
RuntimeError: Torch is not able to use GPU; add --skip-torch-cuda-test to COMMANDLINE_ARGS variable to disable this check

**Reply by greathiago** [reply]
By the way, this is my run.bat file
```
@echo off

set COMMANDLINE_ARGS= --skip-torch-cuda-test
call environment.bat

cd %~dp0webui
call webui-user.bat
```

**Comment by Irmagon** [comment]
> I followed the steps for 2 click installation as written, but receiving that error My GPU is a NVIDIA GTX 1650 with 4 VRAM that works just fine with A1111 (older version). What do I do?

When Forge is installed, downloaded last version torch (cuda121) - this rise "no GPU" error on old vcard

Try install previous version
```
pip uninstall torch torchvision
pip install torch==2.1.2 torchvision==0.16.2 --index-url https://download.pytorch.org/whl/cu118
```

**Comment by spammenotinoz** [comment]
Just tried this project, on a system already running automatic1111 and invokeai, but get the same error with forge.
Tried a 3060ti and 4070ti, both get the same error.
Update: I was able to fix,
- un-installed and re-installed the latest cuda (no-change)
- un-installedpython, deleted the appdata\roaming\python, folder then downloaded and installed the latest python (no-change)
- deleted webui-forge folder, unpacked the install file (no-change)
- ran the upgrade then run (no-change)

Finally,
- deleted webui-forge folder, unpacked the install file (no-change)
Did NOT run upgrade.bat, just the run.bat then it worked.
